src/devices/machine/i3001.h

```diff
@@ -80,9 +80,9 @@ class i3001_device : public device_t
 	virtual void device_start() override;
 
 private:
-	devcb_write_line m_fo_handler;
-	devcb_read8 m_px_handler;
-	devcb_read8 m_sx_handler;
+	device_delegate<void (bool)> m_fo_handler;
+	device_delegate<uint8_t (void)> m_px_handler;
+	device_delegate<uint8_t (void)> m_sx_handler;
```

Comment:
Please use empty parentheses for function argument lists with no arguments, not C-style `(void)`.